src/async/MuxedInterruptDistributor.cc

```diff
@@ -133,4 +138,10 @@ namespace ChimeraTK::async {
     }
   }
 
+  /********************************************************************************************************************/
+  void MuxedInterruptDistributor::activateSubDomain(
+      SubDomain<std::nullptr_t>& subDomain, VersionNumber const& version) {
```

Comment:
Can you add a comment why this function is needed?

Reply:
There is a comment in the API description in the header. It is a virtual function that can be/is overloaded by the implementations to execute extra code. Is that sufficient?

Reply:
yes